plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/LegacyDataHandlerTests.php

```diff
@@ -67,10 +74,13 @@ public function test_post_data_cleanup() {
 
 		// Check counts.
 		$this->assertEquals( 0, $this->sut->count_orders_for_cleanup( array( $orders[0]->get_id() ) ) );
-		$this->assertEquals( 1, $this->sut->count_orders_for_cleanup() );
+		$this->assertEquals( 2, $this->sut->count_orders_for_cleanup() );
 
 		// Confirm that we now have 1 unsynced order (due to the removal of the backup data).
 		$this->assertEquals( 1, wc_get_container()->get( DataSynchronizer::class )->get_current_orders_pending_sync_count() );
+
+		// Cleanup the refund order.
+		$this->sut->cleanup_post_data( $refund_order->get_id() );
```

Comment:
Are there any other asserts that we can make after this final line (about the effects of this final line) other than it not throwing an error? 🤔 E.g., should `count_orders_for_cleanup` be changed?